app/services/filter_tagged_content.rb

```diff
@@ -0,0 +1,62 @@
+class FilterTaggedContent
+  def valid?(document)
```

Comment:
Yeah I get that this is always a personal preference (which is why I haven't actively "requested changes"). Fine to leave.